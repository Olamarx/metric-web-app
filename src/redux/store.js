import { configureStore } from '@reduxjs/toolkit';
import countriesReducer from './Countries';
// import countryDetails, { loadCountry } from './MoreDetails';

const store = configureStore({
  reducer: {
    countriesReducer,
  },
});

export default store;
