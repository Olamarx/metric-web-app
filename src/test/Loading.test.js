import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import Loading from '../components/Loading/Loading';
import store from '../redux/store';

describe('The indivial page of the app countires', () => {
  it('renders properly', async () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Provider store={store}>
          <Loading />
        </Provider>
      </MemoryRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
