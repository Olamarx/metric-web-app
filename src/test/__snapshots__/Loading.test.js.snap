// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The indivial page of the app countires renders properly 1`] = `
<div
  className="ui active dimmer"
>
  <div
    className="ui big text loader"
  >
    Loading...
  </div>
</div>
`;
